component/api/src/main/java/org/exoplatform/social/core/space/SpaceListAccess.java

```diff
@@ -31,12 +31,18 @@
   
   /** The user id. */
   private String userId;
+
+  /** The visited profile id, used while getting list of common spaces between two users. */
+  private String profileUserId;
   
   /** The space filter */
   private SpaceFilter spaceFilter;
   
   /** The type. */
   Type type;
+
+  /** used to specify which constructor is used for commons spaces service */
+  private String methodName;
```

Comment:
This seems not necessary

Reply:
in my case i need to use SpaceListAccess constructor that uses (SpaceStorage spaceStorage, String userId, String profileUserId, Type type) as parameters but i've noticed that there is a constructor with the same parametres and types but with a different body ( See line 161), so i needed to add another parametre to differentiate between usages, (See line 44).

Reply:
So, please change the parameter order to have the otherUserId at the end